third_party/lowrisc/extensions.bzl

```diff
@@ -0,0 +1,19 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def _lowrisc_repos():
+    VERSION = "20240923-1"
+    http_archive(
+        name = "lowrisc_toolchain",
```

Comment:
I think we should still keep the `rv32imcb` in the name

Reply:
Renamed `lowrisc_rv32imcb_toolchain`